package.json

```diff
@@ -16,9 +16,10 @@
     "test:browser": "node ./node_modules/karma/bin/karma start --browsers Chrome",
     "test:webpack": "webpack ./_tests/karma.js ./_tests/bundle.js && mocha --compilers js:babel-core/register ./_tests/bundle.js",
     "test:jest": "NODE_ENV=cjs jest",
-    "build": "npm run build5 && npm run build6",
+    "build": "npm run build5 && npm run build6 && npm run buildwebpack",
     "build5": "BABEL_ENV=cjs babel src -d lib && cp src/index.js.flow lib/index.js.flow",
     "build6": "BABEL_ENV=ejs babel src -d es && cp src/index.js.flow es/index.js.flow",
+    "buildwebpack": "BABEL_ENV=cjs babel webpack -d lib-webpack",
```

Comment:
This is not quite correct way of doing it. 
A bit more right way is:
- move `webpack` dir to `src`
- add `cp lib/webpack ./webpack` as `buildwebpack`

But I would propose just land a local `.eslintrc` with ES5 forced. It's literally a few lines of code to change.

Reply:
The issue is not only about webpack folder - all files in src folder are not IE 11 compatible neither.
Both of the 2 folders need to transform:
* arrow functions
* default value of function parameter

As a result, just changing the code in webpack folder is not enough.

Reply:
About moving `webpack` dir to `src`, there would be an issue:
`es/webpack` would also be in the final bundle - it is a duplicate of `lib/webpack` and is not we actually want..

Reply:
changing eslint, and adding plugins to babelrc should be enought.

Reply:
I see. To make sure we are on the same page, 
* for webpack folder, we do not move it, instead we add eslint rc and change the code to be compatible.
* for src folder, we add the plugins to babelrc

Do they sounds good?

Reply:
Yep. Sounds right.